lib/type/unit/Unit.js

```diff
@@ -1048,7 +1096,7 @@ function factory (type, config, load, typed) {
       'f': {name: 'f', value: 1e-30, scientific: true},
       'a': {name: 'a', value: 1e-36, scientific: true},
       'z': {name: 'z', value: 1e-42, scientific: true},
-      'y': {name: 'y', value: 1e-42, scientific: true}
+      'y': {name: 'y', value: 1e-48, scientific: true}
```

Comment:
Good find!

I'm wondering, now that we have full support derived units, these prefixes of `SQUARED` and `CUBIC` may have become redundant. Maybe we should deprecate `m2` and `m3` some day, just use `m^2` and `m^3` instead.